lib/Doctrine/DBAL/Driver/OCI8/OCI8Statement.php

```diff
@@ -134,7 +141,14 @@ static public function convertPositionalToNamedPlaceholders($statement)
      */
     public function bindValue($param, $value, $type = null)
     {
-        return $this->bindParam($param, $value, $type, null);
+        if ($value instanceof OCI8Descriptor) {
```

Comment:
Here is a full test case using oci8 only

https://gist.github.com/soyuka/16273fde9b00c3c47147b537fc7f27fa

See results without descriptor, binding works but column isn't filled with the value:

```
/home/abluchet/forks/dbal/oci8test.php:50:
array(4) {
  [0] =>
  array(3) {
    'ID' =>
    string(1) "2"
    'ADDRESS' =>
    string(0) ""
    'OTHER' =>
    string(47) "Is using a descriptor with address value "null""
  }
  [1] =>
  array(3) {
    'ID' =>
    string(1) "3"
    'ADDRESS' =>
    NULL
    'OTHER' =>
    string(39) "No descriptor with address value "null""
  }
  [2] =>
  array(3) {
    'ID' =>
    string(1) "0"
    'ADDRESS' =>
    string(5) "tests"
    'OTHER' =>
    string(48) "Is using a descriptor with address value "tests""
  }
  [3] =>
  array(3) {
    'ID' =>
    string(1) "1"
    'ADDRESS' =>
    NULL
    'OTHER' =>
    string(40) "No descriptor with address value "tests""
  }
}
```